mlir/test/Dialect/SCF/for-loop-continuous-peeling.mlir

```diff
@@ -0,0 +1,46 @@
+// RUN: mlir-opt %s -scf-for-loop-continuous-peeling=convert-single-iter-loops-to-if=true -split-input-file | FileCheck %s
+
+#map = affine_map<(d0, d1)[s0] -> (s0, d0 - d1)>
+func.func @foo(%ub: index) -> index {
+  %c0 = arith.constant 0 : index
+  %step = arith.constant 8 : index
+  %0 = scf.for %iv = %c0 to %ub step %step iter_args(%arg = %c0) -> (index) {
```

Comment:
Add another test case so that both `usePowerSplit = false` and `usePowerSplit = true` is exercised.